### PR TITLE
Created calendar file and added it to feature1

### PR DESCRIPTION
**Pull request by iwood55**
## Description

Created calendar feature and new route for it in the navbar. Motivation to create it was for the mentor/mentee dashboards. Will add another commit for underdog devs color scheme.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

https://user-images.githubusercontent.com/81526595/141348031-98de173a-df1f-45c6-9902-439ef35be8f2.mp4

### CODE REVIEW

**Review by JorshuaG — CHANGES_REQUESTED** [review]
Could you submit a Loom video going over the code? Unfamiliar with Ant Design and I think team would benefit from a walkthrough.

**Review by kennymiesner — APPROVED** [review]
Great job on implementing this important piece of the dashboard puzzle. Would love to see how this looks on the frontend.

**Review by taylorkangillescas — APPROVED** [review]
Great job implementing what you research on ant design into your calendar. Excited to implement this into our dashboards

**Comment by johanna-rodriguez** [comment]
Nice! it is good you are using Ant Design library. This is an important piece of the application. Good job.

**Review by JorshuaG — APPROVED** [review]

**Review by tfbielawski — COMMENTED** [review]
Isaac, nice work on the calendar. It's better than the one I was working on! I saw some css code commented out, but I know you said in your walk-through that you plan on doing styling next. Just thought I'd mention that.

File: src/components/common/Calendar.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  const num = getMonthData(value);
+  return num ? (
+    <div className="notes-month">
+      <section>{num}</section>
+      <span>Backlog number</span>
+    </div>
+  ) : null;
+}
+
+const calendarFeature = () => {
```
Comment (paulstgermain) [inline src/components/common/Calendar.js:64]: Component capitalization!

**Review by paulstgermain — CHANGES_REQUESTED** [review]
Looking good here, except for the way you're designating, exporting, and importing your component - the component should be capitalized! If you can make that change, I can merge this PR.

**Review by gamboaldo — APPROVED** [review]
great job building out this calendar with antd

File: src/index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ import Signup from './components/pages/Signup/Signup';
 import Navbar from './components/pages/Navbar/Navbar';
 
 import ExampleFeature from './components/pages/Navbar/NavbarFeatures/ExampleFeature';
+import calendarFeature from './components/common/Calendar';
```
[the remark this answers is not in the document]
Comment (iwood55) [reply src/index.js:26]: Changing capitalization now!

**Review by paulstgermain — APPROVED** [review]

**Pull request merged**